.github/workflows/test_and_build.yml

```diff
@@ -100,10 +100,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: ./scripts/install_zig.sh
-      - run: zig/zig build fuzz_vsr_superblock -- --seed 123
```

Comment:
Should we put tree/forest fuzz in CI too?

Reply:
I'll add the tree fuzzer in a follow-up PR. There is a bug in the forest fuzzer (revealed by the new minimal config) that makes it fail right now though.